sourced/ml/models/df.py

```diff
@@ -24,8 +24,16 @@ def construct(self, docs: int, tokfreqs: Union[Iterable[Dict[str, int]], Dict[st
             df = tokfreqs
         else:
             df = {}
-            for d in tokfreqs:
-                df.update(d)
+            self.quant_map = tokfreqs[-1]
```

Comment:
Why do you need to catch `TypeError`? We should not change this model at all.

Reply:
I don't get you, if you don't want me to touch this model, quant_map will be stored in self._df

Reply:
Exactly